_includes/tutorials/aggregating-sum/ksql/markup/dev/continuous-query.adoc

```diff
@@ -0,0 +1,5 @@
+Since the output looks right, the next step is to make the query continuous. Issue the following to create a new table that is continuously populated by its query:
```

Comment:
Could use a bit of an explainer as to why we use a table here. Links would be great too.